### PR TITLE
Fixed Channel Chats | CurrentlyPlaying Routing Changes | Browsing Screen Redesign

### PR DESCRIPTION
**Pull request by rauchp**

### CODE REVIEW

**Comment by rauchp** [comment]
Browse Screen

File: civilapp/browse/BroadcastsView.js [hunk 1]
```diff
@@ -67,54 +71,70 @@ class BroadcastsView extends React.Component {
...
-        refreshing={this.props.refreshing}
-        onRefresh={this.props.onRefresh}
-        style={styles.container}
-        numColumns={2}
-        data={broadcasts}
-        renderItem={this.renderItem}
-        keyExtractor={(item, index) => item.broadcastId}
-      />
+        <ScrollView
+        refreshControl={
```
Comment (EphemeralX) [inline civilapp/browse/BroadcastsView.js]: Nit: indent

File: civilapp/browse/BroadcastsView.js [hunk 2]
```diff
@@ -67,54 +71,70 @@ class BroadcastsView extends React.Component {
...
+            <RefreshControl
+                refreshing={this.props.refreshing}
+                onRefresh={this.props.onRefresh}
+                />
+            }>
+            <View style={{  paddingLeft: width / 10, paddingTop: 15, paddingBottom: 12.5}}>
+        <Text style={{ color: '#616262', fontSize: 12, }}>{time}</Text>
+        <Text style={{ color: '#fff', fontWeight: 'bold', fontSize: 22, }}>Live Now</Text>
```
Comment (EphemeralX) [inline civilapp/browse/BroadcastsView.js]: Nit: Indent both <Text> lines here

File: civilapp/browse/BroadcastsView.js [hunk 3]
```diff
@@ -67,54 +71,70 @@ class BroadcastsView extends React.Component {
...
+        </View>
+        {broadcasts.map((broadcast)=>{
+            return this.renderBroadcast(broadcast)
+
```
Comment (EphemeralX) [inline civilapp/browse/BroadcastsView.js]: Nit: Empty space, move } up

File: civilapp/browse/BroadcastsView.js [hunk 4]
```diff
@@ -137,20 +158,30 @@ const styles = StyleSheet.create({
...
-    marginTop: 10,
+  leftOverlayText: {
+      flexDirection: 'column',
+      alignItems: 'flex-start',
+      textAlign: 'left'
   },
+  rightOverlayText: {
+      flexDirection: 'row',
+      alignItems: 'flex-end',
+
```
Comment (EphemeralX) [inline civilapp/browse/BroadcastsView.js]: Nit: indent

File: civilapp/browse/BrowseScreen.js [hunk 5]
```diff
@@ -84,9 +85,8 @@ export default class BrowseScreen extends Component {
     }
 
     return (
-      <Container style={styles.container}>
+      <ImageBackground  source={require('../assets/images/gradient.png')} style={styles.container}>
```
Comment (EphemeralX) [inline civilapp/browse/BrowseScreen.js]: Nit: extra space between ImageBackground and source

File: civilapp/components/CustomHeader.js [hunk 6]
```diff
@@ -48,5 +49,16 @@ class CustomHeader extends Component {
...
+    headerStyle: {
+        elevation: 0,
+        backgroundColor: 'transparent',
+        borderBottomWidth: 1.15,
+        borderBottomColor: '#292A2B',
+        marginRight: 20,
+        marginLeft: 20
+    }
 
+
```
Comment (EphemeralX) [inline civilapp/components/CustomHeader.js]: Nit: Empty space

**Pull request merged**